homeassistant/components/recorder/migration.py

```diff
@@ -313,7 +313,7 @@ def _migrate_schema(
 
     for version in range(current_version, end_version):
         new_version = version + 1
-        _LOGGER.info("Upgrading recorder db schema to version %s", new_version)
+        _LOGGER.warning("Upgrading recorder db schema to version %s", new_version)
```

Comment:
We log with warning level when finishing a schema migration step, I think it makes sense to log with the same level wen starting a migration step.